ice-adapter/src/main/java/com/faforever/iceadapter/util/latencyestimation/HttpLatencyEstimator.java

```diff
@@ -0,0 +1,88 @@
+package com.faforever.iceadapter.util.latencyestimation;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.Socket;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionException;
+
+@Slf4j
+public class HttpLatencyEstimator {
+
+//    public static final String HTTP_REQUEST = "GET / HTTP/1.1\n" +
+//            "Host: traefik.%s\n" +
+//            "\n";
+    public static final String HTTP_REQUEST = "GET / HTTP/0.0\n\n";
+
+    public static CompletableFuture<Double> getLatencyManualHttp(String hostname) {
+        return CompletableFuture.supplyAsync(() -> {
+            try {
+                log.info("Estimating latency to {} using http", hostname);
+
+                Socket socket = new Socket(hostname, 80);
```

Comment:
same as here?

Reply:
not sure if that should even stay in there, the http estimator is currently not used, but I didn't want to just throw it away